### DISCUSSION TITLE
how to connect to wifi from live usb

### DISCUSSION THREAD

**Discussion by dcihon** [ravynsoft/ravynos#376] [General]
How do I connect to wifi from live usb?

**Comment by mszoek** [comment]
Hi! Since we have not written a UI for it yet, you can set it up from command line. Basically, `kldload` the appropriate module if it is not already loaded, add lines like this to `/etc/rc.conf`:
```
wlans_iwlwifi0="wlan0"
ifconfig_wlan0="DHCP WPA"
```

then create `/etc/wpa_supplicant.conf` by following the man page. You probably just need SSID and key parameters.

Do `/etc/rc.d/netif restart` and it should come up.
